.github/workflows/wheels.yml

```diff
@@ -0,0 +1,36 @@
+name: Wheels
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
@zou3519 why we need to execute this workflow (upload wheels) on each PR ? Maybe I'm missing something ...

Reply:
I think the upload wheels only uploads the wheels to github actions. We technically don't need to do this on each PR if we want to save bandwidth but it is still a good check that the wheel building command runs without problem